docs/app/user-directory.md

```diff
@@ -2,54 +2,157 @@
 [Item Page](/app/content-items/). Administrators can customize the fields on this page, but the following are available
 by default.
 
-- **First Name** — The given name
-- **Last Name** — The family/surname
-- **Email** — A unique email address
-- **Password** — A hashed system password
-- **Avatar** — An image to represent the user
-- **Location** — The city, country, office, or branch name
-- **Title** — The professional staff title
-- **Description** — A freeform text description
-- **Tags** — Keywords for searchability
-- **Language** — The preferred App language/locale
-- **Theme** — Light or Dark mode (or based on system preferences)
-- **Multi-Factor Authentication** — Configuration for MFA
-- **Email Notifications** - Receive emails for notifications
+### User Details
 
-The sidebar's info component also includes the following readonly details:
+![User Details](https://cdn.directus.io/docs/v9/app-guide/user-directory/user-directory-20220222A/user-details-20220222A.webp)
 
-- **User Key** — The primary key of the user
-- **Last Page** — The last App page accessed by the user
-- **Last Access** — The timestamp of the user's last App or API action
+- **First Name** – The given name.
+- **Last Name** – The family/surname.
+- **Email** – A unique email address.
+- **Password** – A hashed system password.
+- **Avatar** – An image to represent the user.
+- **Location** – The city, country, office, or branch name.
+- **Title** – The professional staff title.
+- **Description** – A freeform text description.
+- **Tags** – Keywords for search-ability.
 
-::: tip System Fields
+### User Preferences
 
-Administrators have access to additional system fields.
+![User Preferences](https://cdn.directus.io/docs/v9/app-guide/user-directory/user-directory-20220222A/user-preferences-20220222A.webp)
+
+- **Language** – The preferred App language/locale.
+- **Theme** – Light or Dark mode (or based on system preferences).
+- **Multi-Factor Authentication** – Configuration for MFA.
+- **Email Notifications** – Receive emails for notifications.
+
+### Admin Options
+
+![Admin Options](https://cdn.directus.io/docs/v9/app-guide/user-directory/user-directory-20220222A/admin-options-20220222A.webp)
+
+- **Status** – Set User status Draft, Invited, Active, Suspended, Archived.
+- **Role** – Defines the User's Role.
+- **Token** – Accepts any string as user access token. At least 19-20 characters recommended, but we give you the tools
+  and flexibility to set this according to you own internal security policies.
+- **Provider** – _read-only:_ SSO provider associated with user. See our built-in [SSO options](/configuration/sso).
+- **External Identifier** – Displays external identifier generated by SSO provider.
+
+### Read-only Info
+
+![User Profile Sidebar Information](https://cdn.directus.io/docs/v9/app-guide/user-directory/user-directory-20220222A/user-profile-sidebar-information-20220222A.webp)
```

Comment:
This screenshot doesn't actually show the timestamp as it should. Is it broken, or has the user you screenshotted simply not used the platform?

Reply:
<img width="1413" alt="Screen Shot 2022-03-08 at 2 45 34 PM" src="https://user-images.githubusercontent.com/62306630/157322115-a1836d5c-2d52-4dc1-bf12-12f732d954e5.png">

This is my account, which I am currently logged in to. (I hijacked Ben's avatar for some doc pics).
I just checked every single user and NONE of them display a timestamp on "Last Access".

Seems like a bug? 🤔 Want me to report it?